plugin.video.srfplaytv/resources/language/English/strings.po

```diff
@@ -0,0 +1,128 @@
+# Kodi Media Center language file
```

Comment:
please move to `resources/language/resources.language.en_gb/strings.po` (new default for language folders since krypton)